### ISSUE TITLE
Add a notebook demonstrating a possible application to a real-life problem

### ISSUE THREAD

**Issue by ekatef** [pypsa-meets-earth/documentation#8]
As discussed with @davide-f, it's crucial to being able demonstrate how can be the model applied to the studies which are relevant to answer some practically relevant questions. Probably, we could design a simple study demonstrating how is it possible with PyPSA-Earth and include it into tutorial/documentation materials

**Comment by pz-max** [comment]
I think to show real-life problems we should guide people to the following "PyPSA" notebooks:
- https://pypsa.readthedocs.io/en/latest/examples-sector_coupling.html
- https://pypsa.readthedocs.io/en/latest/examples-lopf.html#power-system-optimization
- https://pypsa.readthedocs.io/en/latest/examples-basic.html
- https://pypsa.readthedocs.io/en/latest/examples-other.html

Each of these examples can be theoretically modelled in PyPSA-Earth.
I found these examples really impressive in my early days as they demonstrate how powerful the framework is.

**Comment by ekatef** [comment]
@pz-max, as a first step agree that we can take inspiration from the PyPSA notebooks. As tutorial demonstration they are brilliant.

But having data with the global coverage we could define the problem practical enough to give an absolutely persuasive answer on the most sceptical "What for?" questions :)

**Comment by pz-max** [comment]
@AnasAlgarei, can you provide your notebook that you mentioned. Should run for the basic tutorial case -> Nigeria and Benin 
I think what would be good:
- Use PyPSA-Eur tutorial demonstration as fundament -> https://www.youtube.com/watch?v=mAwhQnNRIvs
- Throughout the notebook reference better to the PyPSA documentation. 
- Goal of the notebook is that people understand the [model design](https://pypsa.readthedocs.io/en/latest/design.html) and how people need to work with the [pypsa component documentation](https://pypsa.readthedocs.io/en/latest/components.html)
- Use all functions provided by the new [pypsa-stats feature](https://github.com/PyPSA/PyPSA/pull/457). I think after successful testing, we could contribute a `pypsa/example` for using stats! :)

**Comment by pz-max** [comment]
@ekatef 
> But having data with the global coverage we could define the problem practical enough to give an absolutely persuasive answer on the most sceptical "What for?" questions :)

As Davide mentioned, for the hackathon part the **"how to analyse and modify the network"** was missing. This is the primary goal of the new notebook.

Most of the examples in the PyPSA documentation include a Germany demonstration. So I think there are quite practical anyways. I think the PyPSA examples that I listed above can be linked at the end of the new analysis/modification notebook, to show people that PyPSA can be flexibly adapted for many real-life use-case. This notebook will be used for the tutorial for the ODDP project.

**Comment by AnasAlgarei** [comment]
> @AnasAlgarei, can you provide your notebook that you mentioned. Should run for the basic tutorial case -> Nigeria and Benin 
> I think what would be good:
> - Use PyPSA-Eur tutorial demonstration as fundament -> https://www.youtube.com/watch?v=mAwhQnNRIvs
> - Throughout the notebook reference better to the PyPSA documentation. 
> - Goal of the notebook is that people understand the [model design](https://pypsa.readthedocs.io/en/latest/design.html) and how people need to work with the [pypsa component documentation](https://pypsa.readthedocs.io/en/latest/components.html)
> - Use all functions provided by the new [pypsa-stats feature](https://github.com/PyPSA/PyPSA/pull/457). I think after successful testing, we could contribute a `pypsa/example` for using stats! :)

Of course! it just needs some refining and I will upload it once it's finished, I also need to look into adding the stats feature.
